Resources/Prototypes/_RMC14/Catalog/Cargo/requisitions_catalog.yml

```diff
@@ -146,6 +146,8 @@
         crate: RMCCrateGrenadesHighExplosive
       - cost: 3000
         crate: RMCCrateGrenadesFrag
+      - cost: 2000
+        crate: RMCCrateGrenadesSmoke
```

Comment:
I'd rather follow 13 first and get all grenades ported before we start thinking about letting req buy ones they can't in 13, specially in a PR where the rest of the changes are all parity
I asked the CO's if they know why 13 doesn't let you and got that it's either because they aren't all that useful outside of a couple spec roles, or that people were spamming them in briefing
```suggestion
```